style/main.css

```diff
@@ -120,22 +164,30 @@ fieldset > legend {
     }
 
     .controlgroup {
+        flex-grow: 1;
         display: flex;
-        padding-right: 0.5em;
     }
 
-    .controlgroup .control {
+    .control {
         flex-grow: 1;
     }
 
+    .settings-container .control {
+        flex-grow: 0;
+    }
+
     .optionalContent {
         display: none;
     }
 }
 
-.paper_field::before {
+.write-prompt {
     color: blue;
     font-style: italic;
     text-decoration: underline;
-    content: "write...";
-}
\ No newline at end of file
+    cursor: not-allowed;
```

Comment:
This looks great. It's intuitive enough that I think 'Has pen' should be checked by default so that users aren't confused why their `[field]`s do nothing.

Reply:
That's actually a good point.